tests/unit_tests/test_wave_06.py

```diff
@@ -228,21 +229,22 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert not result
```

Comment:
`swap_best_by_category` should return False if a swap doesn't happen. More than be falsy, `result` is intended to return an actual boolean. So while we typically write something like
```py
    assert not result
```
in regular python, here we might prefer the more explicit check of
```py
    assert result == False
```

Same comment applies to similar assertion written in another test below.